index.html

```diff
@@ -35,7 +31,7 @@ <h1 class="logo" href="#"><a>Ra<span class="title-split">mda.js</span></a></h1>
                 </aside>
                 <section>

```

Comment:
U are able to split `aside` and `section` on two columns in the following way: 30% aside, 70% section via flex properties

![image](https://user-images.githubusercontent.com/44003977/218645383-4ed82a52-a4d8-420b-8e3e-86a93357f6c6.png)